<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>1251</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="painter_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="painter">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="txbRecv">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>670</y>
     <width>1221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>1261</width>
     <height>145</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>超声与电机轴间距</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>X1取料</string>
      </property>
     </widget>
    </item>
    <item row="5" column="11">
     <widget class="QPushButton" name="trigger">
      <property name="text">
       <string>trigger</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="x1_fetch"/>
    </item>
    <item row="1" column="11">
     <widget class="QPushButton" name="saveImg">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLineEdit" name="y_fetch_delta"/>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="x2_target"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>X2取料</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLineEdit" name="y1_target"/>
    </item>
    <item row="5" column="8">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Y放料校正</string>
      </property>
     </widget>
    </item>
    <item row="2" column="12">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="motor_rho"/>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X2归零常量</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Y取料校正</string>
      </property>
     </widget>
    </item>
    <item row="2" column="11">
     <widget class="QCheckBox" name="auto_run">
      <property name="text">
       <string>自动</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X1归零常量</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Y取料</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="x2_fetch"/>
    </item>
    <item row="5" column="9">
     <widget class="QLineEdit" name="y_target_delta"/>
    </item>
    <item row="1" column="6">
     <widget class="QLineEdit" name="x1_start"/>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="x1_target"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>X1放料</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="y1_fetch"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>X2零点距离</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QLineEdit" name="fetch_delta"/>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>X取料校正</string>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QLineEdit" name="y1_start"/>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Y放料</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Y归零常量</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>X2放料</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="x2_rho"/>
    </item>
    <item row="4" column="8">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>X放料校正</string>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <widget class="QLineEdit" name="target_delta"/>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>modbus_delay</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLineEdit" name="modbus_delay"/>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>monitor_delay</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLineEdit" name="x2_start"/>
    </item>
    <item row="2" column="8">
     <widget class="QLineEdit" name="monitor_delay"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>1261</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>摄像头参数:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ComboDevices"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="tbExposure"/>
    </item>
    <item>
     <widget class="QLineEdit" name="tbFrameRate"/>
    </item>
    <item>
     <widget class="QLineEdit" name="tbGain"/>
    </item>
    <item>
     <widget class="QLineEdit" name="frames"/>
    </item>
    <item>
     <widget class="QPushButton" name="bnOpen">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Calibration">
      <property name="text">
       <string>校准</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modbus"/>
    </item>
    <item>
     <widget class="QPushButton" name="modbus_win">
      <property name="text">
       <string>ModbusWin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
